packages/core/src/codewhisperer/activation.ts

```diff
@@ -263,7 +263,8 @@ export async function activate(context: ExtContext): Promise<void> {
         vscode.languages.registerCodeActionsProvider(
             [...CodeWhispererConstants.platformLanguageIds],
             SecurityIssueCodeActionProvider.instance
-        )
+        ),
+        vscode.commands.registerCommand('aws.codeWhisperer.openEditorAtRange', openEditorAtRange)
```

Comment:
This is to let the Amazon Q show the message when it is installed with old aws toolkit. Otherwise, it will fail before even showing that message.

Reply:
Cool. Out of scope for this PR but we should probably wrap it with our Commands.register interface instead.